examples/README.md

```diff
@@ -7,6 +7,9 @@ A collection of example apps built with Relm4.
 ```bash
 git clone https://github.com/Relm4/Relm4.git
 cd Relm4
+mkdir -p ~/.local/share/glib-2.0/schemas/
+ln -s $(pwd)/examples/relm4.example.safeties.gschema.xml ~/.local/share/glib-2.0/schemas/
+glib-compile-schemas ~/.local/share/glib-2.0/schemas/
```

Comment:
I would mention that only the new example needs this. The other examples run fine without this, so we should not bother everyone with this.

Reply:
And what should I do? `gio::Settings` requires a schema, and the easiest way to pass it one is to have a schema in ~/.local/share/glib-2.0/.

Or do you mean that I should put that code somewhere else, and specify that running the `menu_action_and_settings` example requires running that?

Reply:
> Or do you mean that I should put that code somewhere else, and specify that running the menu_action_and_settings example requires running that?

Exactly. You can keep it in the README, but ideally in a separate block with a comment that it is only necessary for the `menu_action_and_settings` example.